extra/1-factorial.js

```diff
@@ -9,7 +9,11 @@
 */
 
```

Comment:
Well done for attempting this - it's a tough one!
A couple of hints:
1. Are you using the `input` parameter in your function?
2. Try stepping through your code with the VsCode debugger, and see if it's working as you expect it to. (let us know if you need help with that 😄 )